src/bugreporter/odemis_bugreporter.py

```diff
@@ -256,18 +256,17 @@ def compress_files(self):
                 models = glob(os.path.join(odemis_config['CONFIGPATH'], '*/*.odm.yaml'))
             files.extend(models)
 
-            # FIXME: the following two are not working properly if glob doesn't sort by time,
-            # which it probably doesn't do.
-
             # Add the latest overlay-report if it's possibly related (ie, less than a day old)
-            overlay_reps = glob(os.path.join(home_dir, 'odemis-overlay-report', '*'))
```

Comment:
The previous form was actually slightly more correct, as the whole point of `os.path.join` is that you don't explicitly write `/` or `\`.
This works fine: `glob(os.path.join(home_dir, 'odemis-overlay-report', '*'))`